app/routes/missions.einstein-probe/route.mdx

```diff
@@ -33,6 +33,13 @@ The [Einstein Probe (EP)](https://ep.bao.ac.cn/ep/) is a mission of the Chinese
 
 **JSON-Serialized GCN Notices Types in GCN Kafka:**
 
+Einstein Probe distributes alerts and localizations detections of gamma-ray transients. These notices are published on the GCN Kafka topic `gcn.notices.einstein_probe.wxt.alert`.
+
+The [GCN schema](/schema/stable/gcn/notices/einstein_probe/wxt/alert.schema.json) and
+example [JSON message](/schema/stable/gcn/notices/einstein_probe/wxt/alert.schema.example.json) files are available to use for [Einstein Probe Schema](/schema/stable/gcn/notices/einstein_probe). See the [Schema Browser](/docs/schema/stable/gcn/notices/einstein_probe/wxt/alert.schema.json) for more information on the properties defined in the schema.
+
+[Detailed Description and Examples](https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices/einstein_probe/wxt)
```

Comment:
This is not a complete sentence.

Reply:
There is a table below it, should n't be complete sentence.
<img width="603" alt="Screenshot 2024-06-26 at 3 31 00 PM" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/103223246/7d448404-4643-495f-886e-5a3022a04b68">

Reply:
Yes, it should be complete sentence.